src/coresyf_image_reclassification/coresyf_image_reclassification.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/python2
+from coresyftools.tool import CoReSyFTool
+
+
+class IncorrectRanges(Exception):
+    """Exception thrown when the text file contains incorrect ranges."""
+    def __init__(self, range_values):
+        message = ('Incorrect range: "{}"! ' +
+                   'Use a text file with valid range format! Example:\n' +
+                   '0 - 5 : 1\n' +
+                   '6 - 9 : 3')
+        super(IncorrectRanges, self).__init__(message.format(range_values))
+
+
+def create_gdal_calc_expression(file_path):
+    '''
+    It opens a text file with the correspondence between the input and output
+    classes and returns a string with a gdal_calc expression representing the
+    logic of the data transformation.
+    :param str file_path: path of the text file.
+    '''
+    with open(file_path) as f:
```

Comment:
Perhaps you should rename the variable for the open file. Something like "input_file".